doc/governance/proposals.rst

```diff
@@ -278,18 +278,6 @@ For example, ``member1`` may submit a proposal to add a new member (``member4``)
       "state": "Open"
     }
 
-Or alternatively, with the old signature method:
```

Comment:
Same here I think, we should add a warning at the top of the page to say that this has been removed in 4.x.